src/labbox-react/MainWindow/addWorkspace.md

```diff
@@ -3,35 +3,11 @@ To add a workspace, run the following Python script on the computer where the ba
 ```python
 import sortingview
 
-# replace "new-workspace" with the name of the new workspace
-new_workspace_name = # 'new-workspace'
+# replace "new-workspace" with the label of the new workspace
+new_workspace_label = # 'new-workspace'
 
-workspace_list = sortingview.WorkspaceList(list_name='default')
-new_workspace = sortingview.create_workspace()
-workspace_list.add_workspace(name=new_workspace_name, workspace=new_workspace)
-```
+new_workspace = sortingview.create_workspace(label=new_workspace_label)
+sortingview.add_workspace_to_list(list_name='default', workspace=new_workspace)
```

Comment:
This is just curiosity--do we ever use a different `list_name`? What happens if we do?

Reply:
Right now list_name is always 'default', but I'm just thinking about a future time when we may want more than one list. Just not sure.